devolutions-gateway/src/config.rs

```diff
@@ -361,10 +361,17 @@ impl WebAppConf {
         } else if let Some(path) = &value.static_root_path {
             path.as_std_path().to_owned()
         } else {
-            let mut exe_path = std::env::current_exe().context("failed to find service executable location")?;
-            exe_path.pop();
-            exe_path.push("webapp");
-            exe_path
+            if cfg!(target_os = "windows") {
+                let mut exe_path = std::env::current_exe().context("failed to find service executable location")?;
+                exe_path.pop();
+                exe_path.push("webapp");
+                exe_path
+            } else {
+                let mut root_path = "/usr/share";
+                root_path.push(APPLICATION_DIR);
+                root_path.push("webapp");
+                root_path
```

Comment:
Since we don't package any of this for macOS and I'm not sure what the proper path would be (not /usr/share).

I think if you currently run the Gateway w/web frontend on macOS, you're expected to configure the path manually.

Reply:
In that case, it would be best to fallback on an error, just to make it clear.

```rust
            if cfg!(target_os = "windows") {
                let mut exe_path = std::env::current_exe().context("failed to find service executable location")?;
                exe_path.pop();
                exe_path.push("webapp");
                exe_path
            } else if cfg!(target_os = "linux") {
                let mut root_path = std::path::PathBuf::from("/usr/share");
                root_path.push(APPLICATION_DIR);
                root_path.push("webapp");
                root_path
            } else {
                 anyhow::bail!("standalone web application path should be specified manually on this platform");
            }
```

Reply:
Done 👍